tools/build_api_docs.py

```diff
@@ -2,30 +2,25 @@
 
 import os
 
-from absl import app
-from absl import flags
+from absl import app, flags
+from tensorflow_docs.api_generator import generate_lib, public_api
 
 import model_card_toolkit
 
-from tensorflow_docs.api_generator import generate_lib
-from tensorflow_docs.api_generator import public_api
-
 FLAGS = flags.FLAGS
 
-flags.DEFINE_string(
-    "output_dir",
-    default="/tmp/model_card_toolkit",
-    help="Where to output the docs")
+flags.DEFINE_string("output_dir",
```

Comment:
I personally find the original way of putting the arguments on a new line easier to read. I'm not familiar enough with yapf to know exactly which rule controls this. Maybe SPLIT_BEFORE_EXPRESSION_AFTER_OPENING_PAREN or SPLIT_BEFORE_FIRST_ARGUMENT?